packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1237,6 +1258,15 @@ async function loadClient<TChannelFactory extends IChannelFactory>(
 	);
 }
 
+function setupFuzzSerializer(
+	channel: IChannel,
+	dataStoreRuntime: MockFluidDataStoreRuntime,
+): void {
+	// TODO: More legitimate way to inject this alternative serialization strategy
```

Comment:
Any thoughts on a more legitimate way to do this? I don't really want to expose serializer more than it already is or change things like SharedObject constructors to allow dependency injecting it.

If we don't come up with something better, I'll at least add some runtime validation here that the channel *had* something that looked like a serializer that we're overwriting here, but even that is pretty ugly...

Reply:
probably not right now. once we close the door on writing dds outside our repo, we'll have significantly more flexiblity in how we implement something like this. maybe add a work item under AB#36246 to fix improve this later.